src/Apps/CollectorProfile/Routes/Saves2/Components/FourUpImageLayout.tsx

```diff
@@ -0,0 +1,73 @@
+import { Box, Flex, Image, Spacer } from "@artsy/palette"
+import { prepareImageURLs } from "Apps/CollectorProfile/Routes/Saves2/Utils/prepareImageURLs"
+import { FC } from "react"
+import { cropped } from "Utils/resized"
+
+interface FourUpImageLayoutProps {
+  imageURLs: (string | null)[]
+}
+
+interface RowImageProps {
+  url: string | null
+}
+
+const LARGE_IMAGE_SIZE = 100
+const SMALL_IMAGE_SIZE = 58
+const IMAGE_OFFSET = "2px"
+
+export const FourUpImageLayout: FC<FourUpImageLayoutProps> = ({
+  imageURLs,
+}) => {
+  const preparedImageURLs = prepareImageURLs(imageURLs)
+
+  return (
+    <Box>
+      <Flex flexDirection="row">
+        <RowImage url={preparedImageURLs[0]} />
+        <Spacer x={IMAGE_OFFSET} />
+        <RowImage url={preparedImageURLs[1]} />
+      </Flex>
+
+      <Spacer y={IMAGE_OFFSET} />
+
+      <Flex flexDirection="row">
+        <RowImage url={preparedImageURLs[2]} />
+        <Spacer x={IMAGE_OFFSET} />
+        <RowImage url={preparedImageURLs[3]} />
+      </Flex>
+    </Box>
+  )
+}
+
+const RowImage: FC<RowImageProps> = ({ url }) => {
+  const SIZE = [SMALL_IMAGE_SIZE, LARGE_IMAGE_SIZE]
+
+  if (url === null) {
+    return (
+      <Box
+        width={SIZE}
+        height={SIZE}
+        bg="black5"
+        border="1px solid"
+        borderColor="black10"
+        aria-label="Image placeholder"
+      />
+    )
+  }
+
+  const image = cropped(url, {
+    width: LARGE_IMAGE_SIZE,
+    height: LARGE_IMAGE_SIZE,
+  })
```

Comment:
**question**: or is it better to query cropped image from backend (see implementation [here](https://github.com/artsy/force/commit/5aa0da21f98805578aab198414c82993780a88eb))?

Reply:
It doesn't make a difference — whatever is easier. In this case, using the `cropped` function is easier.

Reply:
> It doesn't make a difference

oh, good to know. Thanks!